Documentation/dhcp.md

```diff
@@ -33,3 +33,5 @@ With the daemon running, containers using the dhcp plugin can be launched.
 ## Network configuration reference
 
 * `type` (string, required): "dhcp"
+* `sendClientID` (boolean, optional): whether to include client identifier option (61) in messages to the DHCP server. Defaults to true.
```

Comment:
"client identifier" is Capitalized in other file(s) below. Make consistently Capital Phrases for these DHCP Terms of Art.